tests/integration/_tmux_session.py

```diff
@@ -294,7 +294,7 @@ def interaction(
             if showing:
                 if isinstance(search_within_response, str):
                     for line in showing:
-                        if search_within_response in line:
+                        if (showing[0] == "SSH password:") or (search_within_response in line):
```

Comment:
let's do this differently.... we shouldn't need to touch the tmux session file....